docs/projects/fps/tutorial.md

```diff
@@ -108,18 +110,20 @@ The `EntityBuilder` syntax provides a compact way to declare the relevant compon
 
 #### Add components
 
-From your `HealthComponent`, the GDK has generated a `Pickups.HealthPickup.Component.CreateSchemaComponentData()` function. The property numbers in your schema file (i.e. `= 1` and `= 2`) determine the order of properties expected as the function's parameters.
+From your `HealthComponent`, the GDK has generated a `Pickups.HealthPickup.Snapshot` struct.
 
-This component can then be added to the `HealthPickup` entity using the line: `.AddComponent(healthPickupComponent, gameLogic)`. The three "well-known components" (`Position`, `Metadata` and `Persistence`) must appear in that order, but after that you are free to add your remaining components in any order you like. Just remember that to complete the pattern the **final call** must be to `.Build();`.
+To add an instance of this struct to the `HealthPickup` entity use the line: `.AddComponent(healthPickupComponent, gameLogic)`.
 
 #### Set permissions (ACLs)
 
 [Access Control Lists]({{urlRoot}}/content/glossary#access-control-list-acl) are how SpatialOS specifies which workers have permission to read-from or write-to the values of certain components. There may be data which you want to be kept private to server-side workers (because clients might use that knowledge to cheat!). Some components should definitely restrict their write-access to specific workers (e.g. a particular player's client) or to server-side workers only, to prevent exploits. For example, in an RPG a player should probably not be able to update the amount of gold they are carrying (at least, not without the server-side validating they aren't cheating!).
 
-In the EntityBuilder syntax, the `.SetReadAcl(AllWorkerAttributes)` function call stated that all worker types should be able to read the data for this entity.
+In the EntityTemplate syntax, the `.SetReadAccess(gameLogic, WorkerUtils.UnityClient)` function call stated that all worker types should be able to read the data for this entity.
 
 For each of the other components, such as your newly added `HealthPickup` component, the worker type which is given write-access is specified as a second argument to the component-adding function, e.g. `WorkerUtils.UnityGameLogic`. This is simply a string which identifies which worker type should be granted the relevant permission.
 
+The `EntityTemplate` also provides methods for manipulating the `ACL` of an entity without adding a snapshot. For example, `.SetComponentWriteAccess(EntityAcl.Id, gameLogic)` will set the write access for the `EntityAcl` component.
```

Comment:
```suggestion
The `EntityTemplate` also provides methods for manipulating the `ACL` of an entity without adding a snapshot. For example, `.SetComponentWriteAccess(EntityAcl.Id, gameLogic)` sets the write access for the `EntityAcl` component.
```
Don't need predictive future "will" - this always happens so we can use present simple.

Reply:
Done!